assets/js/NationalCalendarPayload.js

```
@@ -130,28 +130,28 @@ class LitCalMoveFestivityData extends LitCalFestivityData {
         if (
-            typeof festivity.day !== 'number'
-            || !Number.isInteger(festivity.day)
-            || false === LitCalMoveFestivityData.#isValidDayValueForMonth(festivity.month, festivity.day)
+            typeof liturgical_event.day !== 'number'
+            || !Number.isInteger(liturgical_event.day)
+            || false === LitCalMoveEventData.#isValidDayValueForMonth(liturgical_event.month, liturgical_event.day)
         ) {
-            throw new Error('`festivity.day` must be an integer between 1 and 31 and it must be a valid day value for the given month');
+            throw new Error('`liturgical_event.day` must be an integer between 1 and 31 and it must be a valid day value for the given month');
         }
         return Object.freeze(this);
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)